src/07_slices.py

```diff
@@ -12,16 +12,17 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print("Second Element:", a[1])
 
 # Output the second-to-last element: 9
-print()
+# print("With Len:", a[len(a)-2])
+print("Second to last:", a[4])
 
 # Output the last three elements in the array: [7, 9, 6]
-print()
+print("Last Three: ", a[3:])
 
 # Output the two middle elements in the array: [1, 7]
-print()
+print("Two Middle: ", a[2:4])
 
 # Output every element except the first one: [4, 1, 7, 9, 6]
 print()
```

Comment:
Missed this one :).

Reply:
Thanks, I missed a few I saw, went back and fixed them